<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeNameSetterDialog</class>
 <widget class="QDialog" name="NodeNameSetterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="styleSheet">
      <string notr="true">QLabel  { 
font-family: &quot;SF Pro Text&quot;;
font-size: 13px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: #DA3135; 
background-color: #FDEBEB; 
font-family: Lato; 
font-size: 12px; 
border-radius: 2px; 
border: 1px solid rgba(218,49,53,0.5); 
padding-top: 3px; 
padding-bottom: 5px; 
padding-left: 13px;
padding-right: 13px; 
}</string>
     </property>
     <property name="text">
      <string>The following characters are not allowed:
%1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit { 
font-family: &quot;SF UI Text&quot;;
font-size: 13px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
